assets/shaders/extended_material.wgsl

```diff
@@ -17,6 +17,8 @@
 
 struct MyExtendedMaterial {
     quantize_steps: u32,
+    // Web examples WebGL2 support: structs must be 16 byte aligned.
+    _webgl2_padding: vec3<f32>
```

Comment:
vec3 itself has 16-byte alignment and size 12. So doing this makes this struct have size 28. Three 32-bit scalars would be a better choice for padding here.